installer/scripts/omsadmin.sh

```diff
@@ -381,18 +398,18 @@ set_omsagent_proc_limit()
 
     log_info "Setting process limit for the $AGENT_USER user in $PROC_LIMIT_CONF to $NEW_OMSAGENT_PROC_LIMIT..."
     local new_omsagent_line="$AGENT_USER  hard  nproc  $NEW_OMSAGENT_PROC_LIMIT"
-    find_current_omsagent_proc_limit
+    get_current_omsagent_proc_limit >/dev/null 2>/dev/null
```

Comment:
Redirecting stdout and stderr is standardized in this file as `> /dev/null 2>&1`; do not use the format `>/dev/null 2>/dev/null`

Reply:
Ok.